lib/levelup.js

```diff
@@ -46,7 +46,7 @@ function LevelUP (db, options, callback) {
   options = options || {}
 
   if (!db || typeof db !== 'object') {
-    error = new InitializationError('Must provide db')
+    error = new InitializationError('The first `db` argument must be an abstract-leveldown compliant store')
```

Comment:
Thanks for picking this up. I think I prefer "First argument must be.." because:

1. `db` is an internal variable, its name doesn't matter to the outside
2. "the first `db` argument" seems to imply there is a second `db` ;)

Reply:
You're right! Now that I reread it again it sounds weird, updating it.